qemu/tests/cfg/block_commit.cfg

```diff
@@ -30,6 +30,11 @@
             virtio_scsi:
                 no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
                 bus_extra_params_image1 = "iothread=iothread0"
+                Windows:
+                    i440fx:
+                        cd_format_cd1 = ide
```

Comment:
If iothread is not supported by all scsi-cd devices, here should be `cd_format = xx`, right?

Reply:
Thanks, your are right,  i use `cd_format_cd1` as there is only cd1 for windows guest.
Considering all cdrom, and no matter linux or windows guest,  we can use "cd_format = ide" for all data plan, will update it, thanks !

Reply:
@luckyh we can not use "cd_dormat" as it will be replaced by guest-hw.cfg.

Reply:
why not fix it in `guest-hw.cfg`

Reply:
@hereischen To define test case level parameters in guest-hw.cfg alike files has already introduced very headache issue for us, considering we might rearrange the order of cfg files in the future, I suppose we'd better define these varaints in the cases themselves. What do you think?